packages/components/src/Combobox/ComboboxProvider.tsx

```diff
@@ -7,6 +7,7 @@ export const ComboboxContext = React.createContext(
     open: boolean;
     setOpen: (open: boolean) => void;
     wrapperRef: RefObject<HTMLDivElement>;
+    subjectNoun?: string;
```

Comment:
do we need this?

I'm not seeing it being passed in nor exposed in the `value`

Reply:
Oops! We do not! Thank you!